api/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <groupId>com.okta.sdk</groupId>
         <artifactId>okta-sdk-root</artifactId>
-        <version>2.0.1-SNAPSHOT</version>
+        <version>2.1.0-SNAPSHOT</version>
```

Comment:
Considering this a `minor` version upgrade since we are adding a backward compatible change (compare to v2.0.0) by introducing a new parent interface `UserFactorProfile`.